app/controllers/network_routes_controller.go

```diff
@@ -562,11 +562,12 @@ func stringSliceB64Decode(s []string) ([]string, error) {
 	ss := make([]string, 0)
 	for _, b64String := range s {
 		decoded, err := base64.StdEncoding.DecodeString(b64String)
+		decodedStr := strings.TrimSuffix(string(decoded), "\n")
```

Comment:
bytes.TrimSpace() ?

Reply:
Space is not a problem. base64 encoding adds a new line. May be i will try 

https://golang.org/pkg/encoding/base64/#Encoding.Decode

Reply:
Kubernetes [advises](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually) to use `echo -n "pasword" | base64` where
```
-n     do not output the trailing newline
```